files/tasks/install-ogr-deps.yaml

```diff
@@ -0,0 +1,28 @@
+# Install dependencies from setup.cfg as rpm packages for ogr
+- name: Check that the setupcfg2rpm.py exists
+  stat:
+    path: ./setupcfg2rpm.py
+  register: setupcf2rpm_stat_result
```

Comment:
Maybe this was introduced in a prev PR, still: wouldn't be better to remove this check and just always download the script? When can it happen that the script is already present, and if so, how is it ensured that it's up to date?

Reply:
both tasks files/tasks/install-ogr-deps.yaml and files/tasks/install-packit-deps.yaml use this script. To avoid unnecessary downloads and related "delays" I decided to add this check. 
Because this is run during build time it will be always the most recent version.

On the other hand, if removed, we will avoid any potential future issues. So will remove it.

Reply:
fixed